package.json

```diff
@@ -14,9 +12,7 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject",
```

Comment:
Why removing the deploy scripts ? we need them

Reply:
098b66e